backend/plugins/authorization.js

```diff
@@ -0,0 +1,68 @@
+var errors = require('restify-errors');
+
+import {getSetting} from '../settings.js';
+import {generateAndSaveAccessToken} from '../utils/authUtils';
+import fetch from 'node-fetch';
+
+const ESCAPED_ROUTES = [
+  new RegExp('^/$'),
+  new RegExp('^/login$'),
+  new RegExp('^/static/'),
+  new RegExp('^/oauth2')
+]
+
+function accessTokenIsValid(access_token) {
+    return getSetting('ACCESS_TOKEN') === access_token;
+}
+
+let win = '';
+
+export function PlotlyOAuth(electronWindow) {
+    win = electronWindow;
+
+    function isAuthorized(req, res, next) {
+        let path = req.href();
```

Comment:
can this be `const`? it doesn't look like it's being reassigned.

we really gotta get `eslint` in our tests.